schema/deploy/computed_columns/application_revision_total_ciip_emissions.sql

```diff
@@ -0,0 +1,24 @@
+-- Deploy ggircs-portal:computed_columns/application_revision_facility_emission to pg
+-- requires: tables/gas_001
+-- requires: views/ciip_emission
+
+begin;
+
+-- Computed column returns the sum of (ciip) emissions for an application revision.
+create or replace function ggircs_portal.application_revision_total_ciip_emissions(application_revision ggircs_portal.application_revision)
+returns table (total_ciip_co2e_emissions numeric, total_ciip_annual_emissions numeric)
```

Comment:
Just return a numeric with the `total_ciip_co2e_emissions`. You cannot do a `sum(annual_emission)`, since those are different gases.